src/patternfly/components/Switch/switch.scss

```diff
@@ -1,12 +1,13 @@
 .pf-c-switch {
   // Switch
-  --pf-c-switch--Height: calc(var(--pf-c-switch__label--FontSize) * var(--pf-c-switch--LineHeight));
-  --pf-c-switch--Width: calc(var(--pf-c-switch--Height) + #{pf-size-prem(2px)} + var(--pf-c-switch__toggle--before--Width));
+  --pf-c-switch--Height: calc(var(--pf-c-switch--FontSize) * var(--pf-c-switch--LineHeight));
+  --pf-c-switch--Width: calc(var(--pf-c-switch--Height) + var(--pf-c-switch__toggle-icon--Offset) + var(--pf-c-switch__toggle--before--Width));
   --pf-c-switch--LineHeight: var(--pf-global--LineHeight--md);
+  --pf-c-switch--FontSize: var(--pf-global--FontSize--sm);
```

Comment:
Should this be `--pf-global--FontSize--sm`? Everything is too small by 2px now :)

Reply:
yep forgot to change it back to md